packages/truffle-decode-utils/src/conversion.ts

```diff
@@ -146,4 +146,72 @@ export namespace Conversion {
       return value;
     }
   }
-}
\ No newline at end of file
+
+  /**
+   * Convert a mapping representation into a JS Map
+   *
+   * Only converts integer types to BN right now, leaving everything else alone
+   */
+  export function toMap({ keyType, members }: any): Map<any, any> {
```

Comment:
I'm confused -- it doesn't look like `toMap` calls `cleanMappings`, so it looks like if you have a mapping to mappings, the inner mappings won't get cleaned.  Am I mistaken?  What's going on here?  It doesn't look like it eventually gets passed to the `object` case.

Reply:
d'oh silly me